H_w 3/test_positive.rb

```diff
@@ -202,10 +202,53 @@ def test_bug_types_isue
     expected_text = 'Bug'
     actual_text = type_issue[index].text
     assert_equal(expected_text, actual_text)
+  end
+
+  def test_create_or_not_bug_issue
+    register_user
+    add_new_project
 
+    x = rand(1..2)
+
+    if x == 1
+      add_new_issue
+      puts"Your random action isn't create a new bug issue"
+     else puts"Your random action is create a new bug issue"
+    end
+
+    @driver.find_element(:class,'my-page').click
+
+    @wait.until {@driver.find_element(:class,'mypage-box').displayed?}
+
+    classes_els = @driver.find_elements(:css,'.subject > a')
+    my_element = classes_els.find {|el| el.text.include? @bug_issue}
+
+    if my_element == nil
+      drop_down_project = @driver.find_element(:id,'project_quick_jump_box')
+      option = Selenium::WebDriver::Support::Select.new(drop_down_project)
+      option.select_by(:text, @project_name)
+      add_new_issue
+      else my_element.click
+    end
+
+    @wait.until {@driver.find_element(:class,'contextual').displayed?}
+    @driver.find_element(:class,'contextual').click
+
+    @wait.until {@driver.find_element(:css,'#users_for_watcher > label:nth-child(1)').displayed?}
```

Comment:
the css placed here has hardcoded index inside 
can it bring us to the errors in future?
if yes we have to fix that now